package.json

```diff
@@ -27,7 +27,7 @@
     "prettier": "prettier --write",
     "prod": "env-cmd -e devProd -r env/.cmdrc.json yarn start",
     "staging": "env-cmd -e devStaging -r env/.cmdrc.json yarn start",
-    "start": "node scripts/start.js",
+    "start": "rm -rf ./node_modules/.cache/babel-loader && node scripts/start.js",
```

Comment:
what happens if we don't do `rm -rf ./node_modules/.cache/babel-loader `

Reply:
Its recommended in the docs for the graphql loader plugin. I think it wont always catch some changes to queries when hot reloading.
https://www.npmjs.com/package/babel-plugin-import-graphql